docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+services:
+   app:
+       build:
+           context: .
+       ports:
+           - "8001:8000"
+       volumes:
+           - ./:/app
+           - my_media:/files/media
+       command: >
+           sh -c "python manage.py migrate &&
+                     python manage.py runserver 0.0.0.0:8000"
+       env_file:
+           - .env
+       depends_on:
+           - db
+
+   db:
+       image: postgres:14-alpine
+       restart: always
+       env_file:
+           - .env
+       ports:
+           - "5433:5432"
+       volumes:
+           - my_db:$PGDATA
```

Comment:
The volume `my_db` is used for the PostgreSQL data directory. Ensure that `$PGDATA` is correctly set in your environment or Dockerfile to avoid issues with data persistence.